src/current/_includes/releases/v23.1/v23.1.12.md

```diff
@@ -0,0 +1,168 @@
+- Fixed a bug where `atttypmod` in `pg_attribute` was not populated for [`TIMESTAMP`](https://www.cockroachlabs.com/docs/v23.1/timestamp) / [`INTERVAL`](https://www.cockroachlabs.com/docs/v23.1/interval) types, which meant that ORMs could not know the precision of these types properly. [#111727][#111727]
+- Fixed a bug in [transaction insight details](https://www.cockroachlabs.com/docs/v23.1/ui-insights-page) where it was possible to see the contention details of other transactions. Now, CockroachDB will only surface contention details for the current transaction. [#111880][#111880]
+- Fixed a bug where `indoption` inside `pg_index` was not properly encoded causing clients to be unable to decode it as `int2vector`. [#111957][#111957]
+- CockroachDB now properly handles RPC failures on writes using the parallel commit protocol that execute in parallel to the commit operation, avoiding incorrect retryable failures and `transaction unexpectedly committed` assertions by detecting when writes cannot be retried idempotently, instead returning an `AmbiguousResultError`. [#111876][#111876]
+- Fixed an issue where the [optimizer](https://www.cockroachlabs.com/docs/v23.1/cost-based-optimizer) fails to honor the `statement_timeout` session setting when generating constrained index scans for queries with large `IN` lists or `= ANY` predicates on multiple index key columns, which may lead to an out of memory condition on the node. [#112076][#112076]
+- Fixed a bug that caused internal errors during query optimization in rare cases. The bug has been present since version v2.1.11, but it is more likely to occur in version v21.2.0 and later, though it is still rare. The bug only presents when a query contains `min` and `max` [aggregate functions](https://www.cockroachlabs.com/docs/v23.1/functions-and-operators#aggregate-functions) [#112254][#112254]
+- Fixed an issue where lookup and index joins on tables with at least 3 column families were sometimes incorrectly evaluated, leading to "non-nullable column with no value" errors or incorrect query results. The bug was introduced in 22.2. [#113107][#113107]
+
+<h3 id="v23-1-12-performance-improvements">Performance improvements</h3>
+
+- Queries that compare collated strings now use less memory and may execute faster. [#110147][#110147]
+- Reduced the impact of high-concurrency blind writes to the same key on goroutine scheduling latency. [#109370][#109370]
+
+<div class="release-note-contributors" markdown="1">
+
+<h3 id="v23-1-12-contributors">Contributors</h3>
+
+This release includes 207 merged PRs by 64 authors.
+We would like to thank the following contributor from the CockroachDB community:
+
+- craig (first-time contributor)
```

Comment:
I don't think this is a real first-time contributor -- can you confirm?

Reply:
@mdlinville I looked at the AUTHORS file and didn't see their name. Is there any other place I should look at?

Reply:
I think "craig" is a bot that we use. https://github.com/cockroachdb/cockroach/pulls/app%2Fcraig

Reply:
ohhh 😄